tests/test_playwright.py

```diff
@@ -692,3 +692,24 @@ def _assert_folder_tree(root: str, expected_tree: List[Any]) -> None:
             assert "children" not in file
         if "children" in file:
             _assert_folder_tree(os.path.join(root, file["name"]), file["children"])
+
+
+def test_is_able_to_set_expect_timeout_via_conftest(testdir: pytest.Testdir) -> None:
+    testdir.makepyfile(
+        """
+        import pytest
+        from playwright.sync_api import expect
+
+        @pytest.fixture(scope="session")
+        def expect_timeout():
```

Comment:
```suggestion
        def expect_options():
```

return dict with timeout as a key.